src/Storefront/Resources/app/storefront/webpack.config.js

```diff
@@ -488,7 +474,7 @@ const mergedCoreConfig = merge([
                             colors: true,
                         },
                     },
-                    hot: false,
+                    hot: true,
```

Comment:
I thought we do not support real hot-reloading because the entire page always makes a page-reload. 

For me this doesn't seem to work when changing CSS in the watcher. The page does not reload and throws a webpack error:

```
Uncaught runtime errors:
×
ERROR
Script error.
    at handleError (webpack-internal:///82199:252:58)
    at eval (webpack-internal:///82199:271:7)
```

Changing JS works fine, also the file names look good / same as in prod.